test/integration/targets/iam_user/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: set up aws connection info
+  set_fact:
```

Comment:
Personally, I have a preference for adding module_defaults to the block below 

``
- name: Test that a user can be created, attached to a group, and then removed
  module_defaults:
    group/aws:
      aws_access_key: "{{ aws_access_key }}"
      aws_secret_key: "{{ aws_secret_key }}"
      security_token: "{{ security_token }}"
      region: "{{ aws_region }}"
  block:
``